### ISSUE TITLE
Newbie questions on using pREST

### ISSUE THREAD

**Issue by annedemata** [prest/prest#290]
- pREST version (or commit ref): 4.0
- pREST endpoint: 
- PostgreSQL version: 9.6.6
- Operating system: Windows 10 (client machine where I have pREST binary) 
- Go version: none
- Log gist: 

## Description
_I posted this same question on Postman forum but maybe I should have posted it here instead ..._
If I’m understanding it correctly, pREST would allow me to work-around Postman’s limitation of not being to directly query a database.

(1) Is my assumption correct that I can install pRest on a Windows client machine and configure it to connect to our PostgreSQL server?

(2) If (1) is correct, is my idea correct that I just need to have 2 files on my working directory - prest-windows-4.0-amd64.exe and prest.toml with my toml file containing the following:

[pg]
host = “127.0.0.1” => replace with PostgresSQL server hostname
user = “postgres” => we’re using this so retain this
pass = “mypass” => replace with postgres password
port = 5432 => retain this
database = “prest” => replace with the database that I’m trying to connect to???

Thanks in advance!!!

PS. I was supposed to post this question on the gitter link provided but I got a 404.

**Comment by avelino** [comment]
@annedemata correct, pREST opens an interface of your PostgreSQL bank as REST API, yes it runs on Microsoft Windows.

2) yeap

**Comment by annedemata** [comment]
Thank you so much for the confirmation, @avelino! 

So I updated my prest.toml accordingly and I also added PREST_DEBUG=true at the top of the file to disable JWT but I'm getting a 401 Unauthorized. I just tried running http://127.0.0.1:6000/databases and http://127.0.0.1:6000/tables. 

The details under [pg] section on prest.toml are the same ones that I'm using successfully on pgAdmin so I'm sure that my connection details are correct. Anything else that I can check? 

If I should create a new thread or if I should post my questions somewhere else, feel free to let me know. 

Thanks again!

**Comment by felipeweb** [comment]
Hi @annedemata 

sorry the docs is a bit outdated.

please add  jwt dafault to false

```toml
[jwt]
 default = false
```
Let me know if you have more doubts.

**Comment by annedemata** [comment]
Yeeeeeeeeeeyyyyyy, that works! 

Thank you, thank you @felipeweb! 
If I can contribute to updating any of the documentations, just let me know (I tried clicking "Edit this page on https://postgres.rest/configurations/ but I got a 404). I'm a tester, though and not a developer so my perspective will be more of a regular user : ).
